README.md

```diff
@@ -219,14 +219,21 @@ If you specify resolvers in a `resolvers.js` file as shown in the quick start ab
 ```yml
 inputs:
   src: ./
-  memory: 512                    # (optional) lambda memory size. default is 3008.
-  timeout: 10                    # (optional) lambda timeout. default is 300.
-  description: My GraphQL App    # (optional) lambda description. default is en empty string.
-  env:                           # (optional) env vars. default is an empty object
-    TABLE: 'my-table'
-  layers:                        # (optional) list of lambda layer arns to attach to your lambda function.
-    - arn:aws:first:layer
-    - arn:aws:second:layer
+  lambdaResolver:                  # (optional) additional parameters for built-in lambda.
+    description: My GraphQL App    # (optional) lambda description. default is en empty string.
+    memory: 512                    # (optional) lambda memory size. default is 3008.
+    timeout: 10                    # (optional) lambda timeout. default is 300.
+    env:                           # (optional) env vars. default is an empty object
+      TABLE: 'my-table'
+    layers:                        # (optional) list of lambda layer arns to attach to your lambda function.
+      - arn:aws:first:layer
+      - arn:aws:second:layer
+    vpcConfig:                     # (optional) specify a vpc
+      securityGroupIds:
+        - sg-xxx
+      subnetIds:
+        - subnet-xxx
+        - subnet-xxx
```

Comment:
Hmm I'm not sure about this. We already have a `resolvers` input where users could specify resolver config, including external lambdas they could use. Having another `lambdaResolver` property sounds confusing to me.

What we are trying to do with the built-in lambda resolver is to close the gap between AppSync and Lambda. To provide an end to end default GraphQL experience that users don't even need to know we are creating a Lambda under the hood. Of course, to configure that build-in lambda, you'd need to be aware of that, but I really think of this lambda as an integrated part of the whole GraphQL component, not a separate thing, which is why I've put all these lambda properties at the top so that you're basically configuring the GraphQL component as a whole, not just the lambda.

Hope that makes sense 😊

Reply:
That makes sense and having the `lambdaResolver` attribute is confusing.

 It does feel very strange to me to have the lambda specific configuration sitting at the top level next to any appsync attributes once you know that you need to set something like `memory` or `vpcConfig` for the built-in resolver, but I don't have a good solution for it hahaha.  Update coming that keeps it flat.

When looking at the inputs without knowing which AWS services accept what it's not intuitive that `timeout`, for instance, is a lambda setting and that appsync uses a default timeout and doesn't accept a custom timeout at this time.